### DISCUSSION TITLE
Symfony 6, Docker, Doctrine/ORM, MongoDB, and V.S. Code using Symfony Binary setup

### DISCUSSION THREAD

**Discussion by Logical-output** [symfony/symfony#50908] [Q&A]
Hey team, I was hoping to get some insight on how to use the Symfony Binary to generate a connection with PostgreSQL, Doctrine/ORM, and Docker using V.S. Code. I read Symfonies Documentation, but it doesn't educate on how to properly use the DATABASE_URL environment variable or the configuration files. I'm not sure how to set them up properly. I also wasn't sure how to use the Symfony binary to connect to PostgreSQL and working with Docker. I also wanted to make sure I can establish a good working local connection. I know the Doc's are meant to be reference material and not hold your hand, but I haven't found a solution and was hoping to get some feedback from the Symfony community... Thanks in advance!

**Comment by tourze** [comment]
I think symfony cli is not for production, only for development.
Dockerize symfony application is same as other dockerize php applicaiton, just write a dockerfile, build image and docker build.
